README.md

```diff
@@ -15,178 +15,122 @@
+We're making it safe and easy for communities to provide mutual aid.
 
-### `npm test`
+MutualAid.world is a global community of contributors making small acts of kindness seamless. We build software to meet the direct needs of local organizers, making it safe and easy for people to help each other during times of crisis.
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+### Built With
 
-### `npm run build`
+- [React.js](https://reactjs.org)
+- [Material UI](https://material-ui.com/)
+- [Typescript](https://www.typescriptlang.org/)
+- [Firebase](https://firebase.google.com/)
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+Here are a few steps to help get set up for local development.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Prerequisites
 
-### `npm run storybook`
+- [Node.js](https://nodejs.org/en/download/)
+- [Npm](https://www.npmjs.com/get-npm) (Comes with Node.js)
+- [Java](https://www.oracle.com/java/technologies/javase-jdk8-downloads.html) (optional)
 
-{DONT THINK WE ARE MAINTAINING THIS CURRENTLY}
+## Repo setup
 
-Run the storybook where we can showcase the components in the system
-[write a story](https://storybook.js.org/docs/basics/writing-stories/)
+1. Fork this repo
+2. Clone the repo
 
-### `npm run docs`
+   ```sh
+   git clone https://github.com/your_username_/Project-Name.git
+   ```
 
-Run this to generate docs in `docs/` folder (made with JSDocs).
-[For more info](https://jsdoc.app/)
+3. Install NPM packages
 
-## Learn More
+   ```sh
+   npm install
+   ```
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+4. Create a local environment file from the sample `.env.sample` in the project root. From your project root run:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+   ```sh
+   cp .env.sample .env.development
+   ```
 
-### Analyzing the Bundle Size
+5. Find the appropriate keys in in the `#resilience-app` channel under `Pinned Items` and fill in your `.env.development`
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
+6. Start your local development server with
 
-## Code change process
+   ```sh
+   npm run dev
+   ```
 
-- See [Contributing.MD](/CONTRIBUTING.md) but... Checkout a new branch by choosing a simple name that expresses the individual issue you're addressing
+   Alternatively if you do not wish to install `java` and use the stage database on firebase, just run
```

Comment:
This language makes it sound like you're free to use either or. While technically true we probably want to more strongly encourage the emulated environment.

In the future I imagine we'll want to only have the emulated env for contributors and then allow direct access to firebase for chosen set of people

Reply:
Cool, can just remove for now